pom.xml

```diff
@@ -87,7 +87,25 @@
           </execution>
         </executions>
       </plugin>
-
+      <plugin>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>unpack-dependencies</id>
+            <phase>prepare-package</phase>
+            <goals>
+              <goal>unpack-dependencies</goal>
```

Comment:
Any specific reason for using unpack-dependencies rather than unpack? My understanding is that this should simplify the configuration, only referencing the groupId/artifactId once. Alternatively maybe do the unpacking through the maven-assembly-plugin so it's only done when actually packaging the project?

https://maven.apache.org/plugins/maven-dependency-plugin/examples/unpacking-artifacts.html
https://maven.apache.org/plugins/maven-dependency-plugin/unpack-mojo.html

Reply:
Switched to `unpack` as suggested. I kept it as part of the `install` goal to make sure local development is still working like before.

Reply:
Side-note (related to the README instructions): `mvn install` is an anti-pattern. Only `package` or `verify` should ever be used (or `deploy` or `clean` of course), possibly `test` in single-module projects (never in reactor builds)

Reply:
Good point, I guess the readme will need a major rewrite if we go with #385 anyway.